### DISCUSSION TITLE
Tell me your hobby favorite sports

### DISCUSSION THREAD

**Discussion by Mont9165** [NAIST-SE/AttractingContributors#1074] [General]
https://github.com/Mont9165/AttractingContributors-hobby-sports

My hobby is basketball

**Comment by satorukano** [comment]
My hobby is judo.

**Comment by Ngochuy2137** [comment]
I have just contributed to your repo. Can you contribute to my repo please https://github.com/Ngochuy2137/AttractingContributors-fruits/tree/main ?

**Comment by anamhamza786** [comment]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git

**Comment by mariawotters** [comment]
Beach volleyball. I also really like yoga and often do it in group classes. It helps reduce my anxiety along with the sedatives from [Canadian pharmacy](https://www.canadapharmacy.com/) . I am very happy that I found sports that make me feel good. It is very cool and I recommend everyone to do sports to feel good and healthy
